test/tests/functional/pbs_soft_walltime.py

```diff
@@ -380,7 +380,7 @@ def test_soft_before_dedicated(self):
         """
 
         now = int(time.time())
-        self.scheduler.add_dedicated_time(start=now + 60, end=now + 2500)
+        self.scheduler.add_dedicated_time(start=now + 80, end=now + 2500)
```

Comment:
@shilpakodli , this is not due to slow machine. If you check before fix result when PTL set 'Resource_List.soft_walltime', dedicated time started. you increased start time from 60 sec. to 80 sec. and if you check dedicated start time not affecting anymore. your test pass because when PTL set 'Resource_List.soft_walltime' dedicated time not started.

Reply:
Thanks Dinesh for catching this. I have addressed your comments. please review.